bitnami/redis/values.yaml

```diff
@@ -916,7 +916,7 @@ sentinel:
   externalMaster:
     enabled: false
     host: ""
-    port: 6379
+    port: 26379
```

Comment:
I believe we need to change this back too. The port is used by sentinel to determine which _redis_ port to monitor.